documentation/cobigen-openapiplugin.asciidoc

```diff
@@ -40,18 +40,18 @@ Additionally, the java plugin provides the ability to match packages (`container
 
 * type 'element'
 
-The container matcher matches elements as Java Objects, in this case will be always an `OpenApiFile` object. (See link:cobigen-core_configuration#containermatcher-node[`containerMatcher` semantics] to get more information about `containerMatchers` itself.)
+The container matcher matches elements as Java Objects, in this case will be always an `OpenAPIFile` object. (See link:cobigen-core_configuration#containermatcher-node[`containerMatcher` semantics] to get more information about `containerMatchers` itself.)
 
-[[openapi-variable-assignment]]
+[[OpenAPI-variable-assignment]]
 === Variable Assignment types
-Furthermore, it provides the ability to extract information from each input for further processing in the templates. The values assigned by variable assignments will be made available in template and the `destinationPath` of context.xml through the namespace `variables.<key>`. The OpenApi Plug-in currently provides two different mechanisms:
+Furthermore, it provides the ability to extract information from each input for further processing in the templates. The values assigned by variable assignments will be made available in template and the `destinationPath` of context.xml through the namespace `variables.<key>`. The OpenAPI Plug-in currently provides two different mechanisms:
 
 * type 'constant' -> constant parameter
 +
 [source,xml]
 ----
 <trigger id="..." type="openapi" templateFolder="...">
-    <containerMatcher type="element" value="OpenApiFile"/>
+    <containerMatcher type="element" value="OpenAPIFile"/>
```

Comment:
this is code, you should not change here anything, please check all occurrences

Reply:
reverted changes